example/mapboxExample.js

```diff
@@ -0,0 +1,176 @@
+import * as THREE from 'three';
+import CameraSync, {
+	projectToWorld,
+	projectedUnitsPerMeter,
+} from './mapboxExampleCamera.js';
+import { DebugTilesRenderer as TilesRenderer } from '../src/index.js';
+import { GUI } from 'three/examples/jsm/libs/lil-gui.module.min.js';
+
+const params = {
+
+	'accessToken': 'pk.eyJ1IjoieGlheGlhbmdmbmVnIiwiYSI6ImNscTRzc245NTA5c3cya3BhdXA4MDY0NWQifQ._9T_J9z7Hvcd00nQL3UuSw',
```

Comment:
It looks like we're still providing a token? I think we should default to a string called "put-your-api-key-here" as the Google Maps example does.

Reply:
ok